src/Parallel/centrality/pagerank.jl

```diff
@@ -8,30 +8,35 @@ function pagerank(
     # indegree(g, v) is estimated run-time to iterate over inneighbors(g, v)
     partitions = LightGraphs.optimal_contiguous_partition(indegree(g), nthreads(), nv(g))
 
-    # collect dangling nodes
-    dangling_nodes = [v for v in vertices(g) if outdegree(g, v) == 0]
+    α_div_outdegree = Vector{Float64}(undef,nv(g))
+    dangling_nodes = Vector{U}()
+    for v in vertices(g)
+        if outdegree(g, v) == 0
+            push!(dangling_nodes, v)
+        end
+        α_div_outdegree[v] = (α/outdegree(g, v))
+    end
+
     N = Int(nv(g))
```

Comment:
Would prefer it you would use lower case `n` or `nvg` instead, if there is no special reason for using `N` (such as already used `n` for something else).